### PR TITLE
[Add-ins] Implement feedback on overview and quick start

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[773b4f1](https://github.com/OfficeDev/outlook-dev-docs/pull/625/commits/773b4f1a6cadee0c7161bb41588f136f3ab30666)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/index.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-sozafar-feedback/docs/add-ins/index.md) | :white_check_mark:Succeeded | |
[docs/add-ins/quick-start.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-sozafar-feedback/docs/add-ins/quick-start.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201906132237556515-625%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=aKi1YLQONQr9MIpYzHaxAVgNim9zUZlYspBP50r0%2BjI%3D&st=2019-06-13T22%3A33%3A52Z&se=2019-07-14T22%3A38%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/index.md [hunk 1]
```diff
@@ -69,7 +69,10 @@ Outlook add-ins are supported in Outlook 2013 or later on Windows, Outlook 2016
 
 ## Get started building Outlook add-ins
 
-To get started building Outlook add-ins, see [Get Started with Outlook add-ins for Office 365](addin-tutorial.md).
+To get started building Outlook add-ins, try the following.
+
+- [Quick start](quick-start.md)
```
Comment (davidchesnut) [inline docs/add-ins/index.md]:
```suggestion
* [Quick start](quick-start.md) - Build a simple task pane.
```

File: docs/add-ins/index.md [hunk 2]
```diff
@@ -69,7 +69,10 @@ Outlook add-ins are supported in Outlook 2013 or later on Windows, Outlook 2016
...
+- [Tutorial](addin-tutorial.md)
```
Comment (davidchesnut) [inline docs/add-ins/index.md]:
```suggestion
* [Tutorial](addin-tutorial.md) - Learn how to create an add-in that inserts gists from GitHub into a new message.
```

File: docs/add-ins/quick-start.md [hunk 3]
```diff
@@ -2,17 +2,17 @@
...
 
 # Build your first Outlook add-in
 
-In this article, you'll walk through the process of building an Outlook task pane add-in.
+In this article, you'll walk through the process of building an Outlook task pane add-in that displays at least one property of a selected message.
 
 ## Create the add-in
 
-You can create an Office Add-in by using Visual Studio or any other editor. Tell us what editor you'd like to use by choosing one of the following tabs:
+You can create an Office Add-in by using Visual Studio or any other editor. Tell us what editor you'd like to use by choosing one of the following tabs.
```
Comment (davidchesnut) [inline docs/add-ins/quick-start.md]:
```suggestion
You can create an Office Add-in by using Visual Studio or any other editor. Tell us which editor you prefer by choosing one of the following tabs.
```

File: docs/add-ins/quick-start.md [hunk 4]
```diff
@@ -159,7 +159,7 @@ When you've completed the wizard, Visual Studio creates a solution that contains
 1. In the **Connect to Exchange email account** dialog box, enter the email address and password for your [Microsoft account](https://account.microsoft.com/account) and then choose **Connect**. When the Outlook.com login page opens in a browser, login to your email account with the same credentials as you entered previously.
 
     > [!NOTE]
-    > If the **Connect to Exchange email account** dialog box repeatedly prompts you to login, Basic Auth may be disabled for accounts on your Office 365 tenant. To test this add-in, login using a [Microsoft account](https://account.microsoft.com/account) instead.
+    > If the **Connect to Exchange email account** dialog box repeatedly prompts you to log in, Basic Auth may be disabled for accounts on your Office 365 tenant. To test this add-in, login using a [Microsoft account](https://account.microsoft.com/account) instead.
```
Comment (davidchesnut) [inline docs/add-ins/quick-start.md]:
```suggestion
    > If the **Connect to Exchange email account** dialog box repeatedly prompts you to sign in, Basic Auth may be disabled for accounts on your Office 365 tenant. To test this add-in, login using a [Microsoft account](https://account.microsoft.com/account) instead.
```
We always use "sign in" and "sign out" unless the dialog really does use the text "log in".

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[893f430](https://github.com/OfficeDev/outlook-dev-docs/pull/625/commits/893f43022b79df79138ec45a5863b3e1a1c0b79f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/index.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-sozafar-feedback/docs/add-ins/index.md) | :white_check_mark:Succeeded | |
[docs/add-ins/quick-start.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-sozafar-feedback/docs/add-ins/quick-start.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C14%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201906142033289373-625%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gtML0lUOYv9fClmTgfbcdmXuWgmtXDGTd9L%2FeAiq4%2Bc%3D&st=2019-06-14T20%3A29%3A22Z&se=2019-07-15T20%3A34%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[55690cc](https://github.com/OfficeDev/outlook-dev-docs/pull/625/commits/55690cc4f8c01106edc0485c5b9259fbc9e9fcd4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/index.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-sozafar-feedback/docs/add-ins/index.md) | :white_check_mark:Succeeded | |
[docs/add-ins/quick-start.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-sozafar-feedback/docs/add-ins/quick-start.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C14%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201906142042266406-625%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=e3OkhqeI3WoluHo912BF33yTbvp5TFmoyIii2fWIgLQ%3D&st=2019-06-14T20%3A38%3A22Z&se=2019-07-15T20%3A43%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**